test/libyul/yulSyntaxTests/call_intruction_in_eof.yul

```diff
@@ -0,0 +1,13 @@
+object "a" {
+    code {
+        let success := call(gas(), 0x1, 0, 128, 4, 128, 0)
+    }
+}
+// ====
+// EVMVersion: >=prague
+// compileTo: >=1
```

Comment:
Any reason not to call this `EOFVersion`? Is that because it looks weird when the value is `legacy`? Then I'd at least give it a less generic name:

```suggestion
// bytecodeFormat: >=EOFv1
```
Could also be `containerFormat`, but IMO `EOFVersion` is perfectly fine too and probably easier to relate to what it means internally.

Reply:
Yes `EOFVersion` looks weird to me when I put `legacy`. but `bytecodeFormat` looks good.

Reply:
I'd still prefer to have `EOF` somewhere in there though. I think that `bytecodeFormat: >=1` is not self explanatory enough. Why not use the one I suggested above?

Reply:
Sure. Just haven't noticed that you added EOFv1 :)